apps/eslint/index.js

```diff
@@ -0,0 +1,63 @@
+module.exports = {
+  rules: {
+    'style-blocks-below-class': {
+      meta: {
+        fixable: 'code'
+      },
+      create: function(context) {
+        return {
+          Program(node) {
+            const styleBlock = node.body.filter(
+              item =>
+                item.type === 'VariableDeclaration' &&
+                item.declarations[0].id.name === 'styles'
```

Comment:
i think this will catch ~90% of style blocks, but there are some throughout apps/ that aren't named 'styles'. what do you think about sending an email to dev@ after this is merged just to 1) let everyone know about this change, and 2) ask folks to rename or manually move any styles that are named differently?

Reply:
Totally! I think there are a lot of cases that this will miss, if we decide to apply this rule, I'll definitely communicate it out.